apis/kubedb/v1alpha2/pgbouncer_types.go

```diff
@@ -118,10 +118,17 @@ const (
 )
 
 type Databases struct {
+	// SyncUsers is a boolean type and when enabled, operator fetches all users created in the backend server to the
+	// PgBouncer server. Password changes are also synced in pgBouncer when it is enabled.
```

Comment:
fetches Postgres users from externally managed secrets and sync them to PgBouncer.

Reply:
@MobarakHsn I got this comment from pgpool though. I already replaced the comment. Can you review it again if it's ok?

Reply:
It was previous mechanism for Pgpool now it works differently.